docs/doxygen/Doxyfile

```diff
@@ -38,7 +38,8 @@ PROJECT_NAME           = "F´ Flight Software - C/C++ Documentation"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = NASA-v1.6.0
+# commented out to avoid having to update it every release
+# PROJECT_NUMBER         = NASA-v1.6.0
```

Comment:
Is there a better way to fill this in?

Reply:
Now using `VERSIONED_OUTPUT` variable, defaults to `devel` if not specified.